cider-velox/src/CiderVeloxPluginCtx.cpp

```diff
@@ -37,10 +37,7 @@ VeloxPlanNodePtr CiderVeloxPluginCtx::transformVeloxPlan(VeloxPlanNodePtr origin
 }
 
 void CiderVeloxPluginCtx::registerTranslator() {
-  static std::once_flag flag;
-  std::call_once(flag, []() {
-    exec::Operator::registerOperator(std::make_unique<CiderPlanNodeTranslator>());
-  });
+  exec::Operator::registerOperator(std::make_unique<CiderPlanNodeTranslator>());
```

Comment:
Q: Do we still need to ensure calling this once within this method?

Reply:
no need ensure
this function will only call once in global initialization.